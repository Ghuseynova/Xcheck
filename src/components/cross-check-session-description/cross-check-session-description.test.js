import React from 'react';
import {render} from "@testing-library/react";
import CrossCheckSessionDescriptionCreation from './cross-check-session-description';

const initialValues = {
  id: "xcheck-1",
  title: "rss2020Q3react-songbird",
  author: "viktorsipach",
  state: "draft",
  taskTitle: "Task 1",
  crossCheckSessionPeriod: [
    "2020-07-08",
    "2020-07-14"
  ],
  coefficient: 0.2,
  discardMinScore: true,
  discardMaxScore: true,
  minReviewsAmount: 1,
  desiredReviewsAmount: 2,
  attendees: []
}

describe('render', () => {    
  test('CrossCheckSessionDescriptionCreation', () => {
  const result = render(<CrossCheckSessionDescriptionCreation descriptionValues={initialValues}/>);
    expect(result).toMatchSnapshot()
  });
});