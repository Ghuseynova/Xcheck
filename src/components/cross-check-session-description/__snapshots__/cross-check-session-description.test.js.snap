// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render CrossCheckSessionDescriptionCreation 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="ant-descriptions ant-descriptions-bordered"
      >
        <div
          class="ant-descriptions-view"
        >
          <table>
            <tbody>
              <tr
                class="ant-descriptions-row"
              >
                <th
                  class="ant-descriptions-item-label"
                  colspan="1"
                >
                  Author
                </th>
                <td
                  class="ant-descriptions-item-content"
                  colspan="1"
                >
                  viktorsipach
                </td>
                <th
                  class="ant-descriptions-item-label"
                  colspan="1"
                >
                  Coefficient
                </th>
                <td
                  class="ant-descriptions-item-content"
                  colspan="1"
                >
                  0.2
                </td>
              </tr>
              <tr
                class="ant-descriptions-row"
              >
                <th
                  class="ant-descriptions-item-label"
                  colspan="1"
                >
                  Start date
                </th>
                <td
                  class="ant-descriptions-item-content"
                  colspan="1"
                >
                  2020-07-08
                </td>
                <th
                  class="ant-descriptions-item-label"
                  colspan="1"
                >
                  End date
                </th>
                <td
                  class="ant-descriptions-item-content"
                  colspan="1"
                >
                  2020-07-14
                </td>
              </tr>
              <tr
                class="ant-descriptions-row"
              >
                <th
                  class="ant-descriptions-item-label"
                  colspan="1"
                >
                  Task
                </th>
                <td
                  class=""
                  colspan="3"
                />
              </tr>
              <tr
                class="ant-descriptions-row"
              >
                <th
                  class="ant-descriptions-item-label"
                  colspan="1"
                >
                  Attendees
                </th>
                <td
                  class="ant-descriptions-item-content"
                  colspan="3"
                >
                  <div>
                    <span
                      style="font-weight: 700; margin-right: 1rem;"
                    >
                      viktorsipach
                    </span>
                    [
                    <span
                      style="margin-right: 1rem; margin-left: 1rem;"
                    >
                      Kipris
                    </span>
                    <span
                      style="margin-right: 1rem; margin-left: 1rem;"
                    >
                      Ghuseynova
                    </span>
                    ]
                    <br />
                  </div>
                  <div>
                    <span
                      style="font-weight: 700; margin-right: 1rem;"
                    >
                      PavelZaharov1987
                    </span>
                    [
                    <span
                      style="margin-right: 1rem; margin-left: 1rem;"
                    >
                      viktorsipach
                    </span>
                    <span
                      style="margin-right: 1rem; margin-left: 1rem;"
                    >
                      Ghuseynova
                    </span>
                    ]
                    <br />
                  </div>
                  <div>
                    <span
                      style="font-weight: 700; margin-right: 1rem;"
                    >
                      Ghuseynova
                    </span>
                    [
                    <span
                      style="margin-right: 1rem; margin-left: 1rem;"
                    >
                      Kipris
                    </span>
                    <span
                      style="margin-right: 1rem; margin-left: 1rem;"
                    >
                      PavelZaharov1987
                    </span>
                    ]
                    <br />
                  </div>
                  <div>
                    <span
                      style="font-weight: 700; margin-right: 1rem;"
                    >
                      Kipris
                    </span>
                    [
                    <span
                      style="margin-right: 1rem; margin-left: 1rem;"
                    >
                      viktorsipach
                    </span>
                    <span
                      style="margin-right: 1rem; margin-left: 1rem;"
                    >
                      PavelZaharov1987
                    </span>
                    ]
                    <br />
                  </div>
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="ant-descriptions ant-descriptions-bordered"
    >
      <div
        class="ant-descriptions-view"
      >
        <table>
          <tbody>
            <tr
              class="ant-descriptions-row"
            >
              <th
                class="ant-descriptions-item-label"
                colspan="1"
              >
                Author
              </th>
              <td
                class="ant-descriptions-item-content"
                colspan="1"
              >
                viktorsipach
              </td>
              <th
                class="ant-descriptions-item-label"
                colspan="1"
              >
                Coefficient
              </th>
              <td
                class="ant-descriptions-item-content"
                colspan="1"
              >
                0.2
              </td>
            </tr>
            <tr
              class="ant-descriptions-row"
            >
              <th
                class="ant-descriptions-item-label"
                colspan="1"
              >
                Start date
              </th>
              <td
                class="ant-descriptions-item-content"
                colspan="1"
              >
                2020-07-08
              </td>
              <th
                class="ant-descriptions-item-label"
                colspan="1"
              >
                End date
              </th>
              <td
                class="ant-descriptions-item-content"
                colspan="1"
              >
                2020-07-14
              </td>
            </tr>
            <tr
              class="ant-descriptions-row"
            >
              <th
                class="ant-descriptions-item-label"
                colspan="1"
              >
                Task
              </th>
              <td
                class=""
                colspan="3"
              />
            </tr>
            <tr
              class="ant-descriptions-row"
            >
              <th
                class="ant-descriptions-item-label"
                colspan="1"
              >
                Attendees
              </th>
              <td
                class="ant-descriptions-item-content"
                colspan="3"
              >
                <div>
                  <span
                    style="font-weight: 700; margin-right: 1rem;"
                  >
                    viktorsipach
                  </span>
                  [
                  <span
                    style="margin-right: 1rem; margin-left: 1rem;"
                  >
                    Kipris
                  </span>
                  <span
                    style="margin-right: 1rem; margin-left: 1rem;"
                  >
                    Ghuseynova
                  </span>
                  ]
                  <br />
                </div>
                <div>
                  <span
                    style="font-weight: 700; margin-right: 1rem;"
                  >
                    PavelZaharov1987
                  </span>
                  [
                  <span
                    style="margin-right: 1rem; margin-left: 1rem;"
                  >
                    viktorsipach
                  </span>
                  <span
                    style="margin-right: 1rem; margin-left: 1rem;"
                  >
                    Ghuseynova
                  </span>
                  ]
                  <br />
                </div>
                <div>
                  <span
                    style="font-weight: 700; margin-right: 1rem;"
                  >
                    Ghuseynova
                  </span>
                  [
                  <span
                    style="margin-right: 1rem; margin-left: 1rem;"
                  >
                    Kipris
                  </span>
                  <span
                    style="margin-right: 1rem; margin-left: 1rem;"
                  >
                    PavelZaharov1987
                  </span>
                  ]
                  <br />
                </div>
                <div>
                  <span
                    style="font-weight: 700; margin-right: 1rem;"
                  >
                    Kipris
                  </span>
                  [
                  <span
                    style="margin-right: 1rem; margin-left: 1rem;"
                  >
                    viktorsipach
                  </span>
                  <span
                    style="margin-right: 1rem; margin-left: 1rem;"
                  >
                    PavelZaharov1987
                  </span>
                  ]
                  <br />
                </div>
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
